DEPENDENCIES.md

```diff
@@ -0,0 +1,23 @@
+Dependencies Notes
```

Comment:
Please remove this file and add the comment to the composer.JSON with the reason why you hardcoded the version and the link to the gherkin GitHub issue